apps/desktop/src/lib/utils/url.ts

```diff
@@ -39,3 +41,30 @@ export function remoteUrlIsHttp(url: string): boolean {
 
 	return httpProtocols.includes(gitRemote.protocol);
 }
+
+export interface EditorPathParams {
+	schemeId: string;
+	path: string[];
+	searchParams?: Record<string, string>;
+	line?: number;
+	column?: number;
+}
+
+export function getEditorPath(params: EditorPathParams): string {
```

Comment:
Would this maybe be more appropriately named `getEditorUri`?

Reply:
Agreed